.gitignore

```diff
@@ -1,4 +1,5 @@
 public/
```

Comment:
FYI: As it does not make sense to commit stuff from the _public_ folder (which is automatically generated when hugo builds the doc) it is already part of the .gitignore list. That's why you were lucky and couldn't commit the images you added to public. :)